docs/cloud-tutorials/best-practices.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 8
+title: Best practices
+description: Tips and tricks for succesfully using DBOS Cloud
+---
+
+This section dives into specific behaviors and corner cases you might encounter when using DBOS Cloud.
```

Comment:
Mention "best practices" so it sounds more positive.

In general, I hope this docs can guide people into the right place in a more positive tone.